lib/constants.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    OVERRIDE_DIR: 'overrides',
```

Comment:
Moving to a constants file makes sense when they're used in multiple places, I think they're only in index.js right?

Reply:
If we move to saving in the legacy generation they'll be used elsewhere. I think at least GENERATED is already used in a couple files.